upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -120,3 +120,11 @@ There you can find links to upgrade notes for other versions too.
         - `Shopsys\ReadModelBundle\Product\Listed\ListedProductViewFacade::getIdsForProducts()`
             use `Shopsys\ReadModelBundle\Product\Listed\ListedProductViewFactory::getIdsForProducts()` instead
     - see #project-base-diff to update your project
+
+- added more coding standards ([#2035](https://github.com/shopsys/shopsys/pull/2035))
+    - the most of the rules have their own fixer, run `php phing ecs-fix` to resolve them 
```

Comment:
I would mention the need of running `standards-fix` several times, so user is not surprised (I was 😆 )